src/main/java/Main.java

```diff
@@ -0,0 +1,56 @@
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import service.CsvFileReaderService;
+import service.CsvFileWriterService;
+import service.FruitService;
+import service.FruitTransactionParser;
+import service.ReportGenerator;
+import service.impl.CsvFileReaderServiceImpl;
+import service.impl.CsvFileWriterServiceImpl;
+import service.impl.FruitServiceImpl;
+import service.impl.FruitTransactionParserImpl;
+import service.impl.ReportGeneratorImpl;
+import strategy.OperationHandler;
+import strategy.handlers.BalanceHandler;
+import strategy.handlers.PurchaseHandler;
+import strategy.handlers.ReturnHandler;
+import strategy.handlers.SupplyHandler;
+
+public class Main {
+    private static final String REPORT_HEADER = "fruit,quantity";
+
+    public static void main(String[] args) {
+        if (args.length != 2) {
+            throw new IllegalArgumentException("Please provide input and "
+                    + "output file paths as arguments.");
+        }
+
+        String inputFilePath = args[0];

```

Comment:
```suggestion
        String inputFilePath = args[INPUT_FILE_PATH_ARG_INDEX];
```
and same for the second one